README.md

```diff
@@ -61,7 +61,7 @@ You may also want to check out [Meilisearch 101](https://www.meilisearch.com/doc
 
 ## ⚡ Supercharge your Meilisearch experience
 
-Say goodbye to server deployment and manual updates with [Meilisearch Cloud](https://www.meilisearch.com/pricing?utm_campaign=oss&utm_source=engine&utm_medium=meilisearch). No credit card required.
+Say goodbye to server deployment and manual updates with [Meilisearch Cloud](https://www.meilisearch.com/cloud?utm_campaign=oss&utm_source=github&utm_medium=meilisearch). No credit card required.
```

Comment:
@Strift to let you know the 30 READMEs in the integration scope do not follow this convention and have `integration` instead of `github`

Reply:
I agree with your choice, my message was mostly to let you know this is not what has been implemented in 95% of the current READMEs 😅